src/components/dataset/details/DatasetDetailView.jsx

```diff
@@ -1,81 +1,105 @@
+    margin: '1rem 0.5rem',
   },
   headerText: {
     fontWeight: theme.typography.bold,
     textTransform: 'uppercase',
-    margin: `${theme.spacing(2)}px ${theme.spacing(1)}px`,
+  },
+  infoColumn: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  infoColumnPanel: {
+    flexGrow: 1,
+  },
+  mainColumn: {
+    display: 'flex',
+    flexDirection: 'column',
+    paddingLeft: '0.5rem',
+  },
+  mainColumnRow1: {},
+  mainColumnRow2: {},
+  mainColumnRow3: {
+    flexGrow: 1,
+  },
+  spacer: {
+    height: 64,
   },
 });
 
-class DatasetDetailView extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      creatingSnapshot: false,
-    };
-  }
-
-  static propTypes = {
-    classes: PropTypes.object,
-    dataset: PropTypes.object,
-    datasetPolicies: PropTypes.array,
-    dispatch: PropTypes.func.isRequired,
-    match: PropTypes.object,
-  };
+const DatasetDetailView = ({
+  classes,
+  dataset,
+  datasetPolicies,
+  dispatch,
+  match: {
+    params: { uuid },
+  },
+}) => {
+  const [creatingSnapshot, setCreatingSnapshot] = useState(false);
 
-  componentDidMount() {
-    const { match, dispatch } = this.props;
-    const { uuid } = match.params;
+  useEffect(() => {
     dispatch(getDatasetById(uuid));
     dispatch(getDatasetPolicy(uuid));
-  }
-
-  openSnapshotCreation = (isOpen) => {
-    this.setState({ creatingSnapshot: isOpen });
-  };
+  }, []);
 
-  render() {
-    const { classes, dataset, datasetPolicies, match } = this.props;
-    const { creatingSnapshot } = this.state;
-    const { uuid } = match.params;
-    if (dataset && datasetPolicies && dataset.id === uuid) {
-      return (
-        <div className={classes.root}>
-          <div className={classes.headerText}>Dataset Information</div>
-          <DatasetInfoCard
-            dataset={dataset}
-            datasetPolicies={datasetPolicies}
-            openSnapshotCreation={this.openSnapshotCreation}
-          />
-          <CreateFullSnapshotView
-            open={creatingSnapshot}
-            openSnapshotCreation={this.openSnapshotCreation}
-          />
+  return datasetPolicies && dataset && dataset.id === uuid ? (
+    <Fragment>
+      <h2 style={{ margin: '0.5rem' }}>{dataset.name}</h2>
+      <div className={classes.root}>
+        <div className={classes.infoColumn}>
+          <div className={classes.headerText}>Dataset Schema (default view)</div>
+          <div className={classes.infoColumnPanel}>
+            <DatasetRelationshipsPanel dataset={dataset} />
+          </div>
+        </div>
+        <div className={classes.mainColumn}>
+          <div className={classes.mainColumnRow1}>
+            <div className={classes.datasetInfoContainer}>
+              <div className={classes.headerText}>Dataset Information</div>
+            </div>
+          </div>
+          <div className={classes.mainColumnRow2}>
+            <DatasetInfoCard dataset={dataset} datasetPolicies={datasetPolicies} />
```

Comment:
You probably don't want to be passing these in.  You should read these from global state in the DatasetInfoCard component